src/core/Akka/Actor/ActorRef.cs

```diff
@@ -76,30 +76,29 @@ public class FutureActorRef : MinimalActorRef
         /// TBD
```

Comment:
This class is simplified-the only purpose of this class now is to listen for the response message and put it into the completion source. Less callbacks, more readable code.